tests/index.js

```diff
@@ -16,3 +21,4 @@ require('./testExam');
 require('./testAccess');
 require('./testZoneGradingHomework');
 require('./testZoneGradingExam');
+require('./testSprocs');
```

Comment:
I really like the addition of sproc unit tests!

One small request: I think it might be cleaner if we call this file `testAccessSprocs` or something, and then in the future when we add more sproc testing we can put it in separate files. This file is already fairly large and complex.